com.unity.multiplayer.mlapi/Runtime/Profiling/ProfilerTickData.cs

```diff
@@ -52,18 +52,19 @@ public void SerializeToStream(Stream stream)
         /// <summary>
         /// Creates a ProfilerTick from data in the provided stream
         /// </summary>
+        /// <param name="manager">The NetworkingManager this ProfilerTick is operating under.</param>
         /// <param name="stream">The stream containing the ProfilerTick data</param>
         /// <returns>The ProfilerTick with data read from the stream</returns>
-		public static ProfilerTick FromStream(Stream stream)
+		public static ProfilerTick FromStream(NetworkingManager manager, Stream stream)
```

Comment:
NM is being passed in so that ProfilerTickData can get an NM so that it can get a PooledBitReader, which - not to belabor the point - should be detached from NM